src/views/v2/rsvp/ari/form/fields.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders the RSVP AR form fields.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/fields.php
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ * @var int $post_id The post ID the RSVP is linked to.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+$has_meta = get_post_meta( $rsvp->ID, '_tribe_tickets_meta_enabled', true );
+
+if ( empty( $has_meta ) ) {
+	return;
+}
+?>
+<div class="tribe-tickets__form">
+
+	<?php $this->template( 'v2/rsvp/ari/form/error', [ 'rsvp' => $rsvp ] ); ?>
+
+	<?php
+		/**
+		 * Allows injection of meta fields in the RSVP ARI form.
+		 *
+		 * @since TBD
+		 *
+		 * @var int $post_id The post ID the RSVP is linked to.
+		 * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+		 */
+		do_action( 'tribe_tickets_rsvp_attendee_fields', $post_id, $rsvp );
```

Comment:
It looks like you removed the component templates and passing the data to them. How is the data being passed to the template? Are we not passing down attributes from a controller to the views?

Reply:
So, I discussed with Scott that we should componetize the meta fields (right now they're all around), and as they belong to ET+, I moved things there: https://github.com/moderntribe/event-tickets-plus/pull/1011/files#diff-653ab140dd2fdeed0daae5941ff48aa7R87

Added a template class, and there's an injection via this hook. You can find it here: https://github.com/moderntribe/event-tickets-plus/pull/1011/files#diff-653ab140dd2fdeed0daae5941ff48aa7R87

Reply:
Gotcha, thanks!